roles/matrix-bridge-mautrix-signal/defaults/main.yml

```diff
@@ -130,6 +130,10 @@ matrix_mautrix_signal_configuration_extension: "{{ matrix_mautrix_signal_configu
 # You most likely don't need to touch this variable. Instead, see `matrix_mautrix_signal_configuration_yaml`.
 matrix_mautrix_signal_configuration: "{{ matrix_mautrix_signal_configuration_yaml|from_yaml|combine(matrix_mautrix_signal_configuration_extension, recursive=True) }}"
 
+# Prevents the puppet from breaking when the signal safety number changes.
+# The new safety number will be marked as trusted_unverified if this is set to true
+matrix_mautrix_signal_daemon_trust_new_keys: false
```

Comment:
This PR is named "Auto trust new signal identities", but the `matrix_mautrix_signal_daemon_trust_new_keys` variable is defaulted to `false`.

Should we rename the PR? Or perhaps switch the default to `true`? I think our conversation was hinting that this shouldn't be set to `true` by default, but.. I'm not really sure what we settled on.

Reply:
The default behavior of the bridge is to have this option disabled. The official Signal programs behave as if this is set to true.
When this option is enabled, it significantly increases the comfort.

If paranoid people want to verify the safety numbers, they can still do that even if the option is on. 
When the option is off, the bridge will no longer send messages to a person when the security number of that person changes, until the safety number is marked manually as verified.

I would prefer the option to be on by default, but I am also happy with off as well.

We could rename the PR to Added option to accept changed signal keys without verifying them manually

Unfortunately, the synonymous use of key, safety number and identity is confusing and is caused by the signal folks.
This thing is something like an individual public key for each conversation and changes every time when a person installs Signal again or linked the Signal Account to a second device (signal desktop, this bridge).